### ISSUE TITLE
Can we redefine this formula using the new neighborhood keyword?

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/PK-Sim#2602]
![image](https://user-images.githubusercontent.com/1041237/233466033-984a8f63-01f2-40e2-b89f-870c72e12714.png)

@Yuri05 @PavelBal  That would be a good use case right?

**Comment by msevestre** [comment]
@Yuri05 BTW: Are those parameters really individual parameters?
![image](https://user-images.githubusercontent.com/1041237/233466473-73dd54e3-c6f5-433b-9fcb-ab70645ed3f4.png)

**Comment by Yuri05** [comment]
Yep, we should redefine lot of formulas. 
Especially those in the calculation methods, which are currently pointing to nbhoods

**Comment by Yuri05** [comment]
>Are those parameters really individual parameters?

Yes. Calculation depends only on other individual parameters and does not depend on the model structure.
